frontend-service/src/main/java/com/amazonaws/blox/frontend/operations/DescribeEnvironment.java

```diff
@@ -35,6 +38,15 @@
 
   @RequestMapping(path = "/{environmentName}", method = RequestMethod.GET)
   @ApiOperation(value = "Describe an Environment by name")
+  @ApiResponses(
+    value = {
+      @ApiResponse(
+        code = 404,
+        message = "Resource not found",
```

Comment:
It might be worth extracting the code and message as constants on the ResourceNotFoundExceptionResponse class.

Reply:
Good point, thanks, will update.